tests/test_random_order.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from copy import deepcopy
+
+from parameterized import parameterized
+
+from monai.data import MetaTensor
+from monai.transforms import EnsureChannelFirst, InvertibleTransform, RandomOrder, TraceableTransform, Transform
+from monai.transforms.compose import Compose
+from monai.transforms.transform import MapTransform
+
+
+class X(Transform):
```

Comment:
I think these class test cases could be imported from test_one_of.py if possible.

Reply:
Thanks! I will update it!